src/Revit/RevitHyparTools/RevitExtensions.cs

```diff
@@ -1,11 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using Autodesk.Revit.DB;
 using Elements.Geometry;
 
-namespace Hypar.Revit 
+namespace Hypar.Revit
 {
-    public static class RevitExtensions {
-        public static Vector3 ToVector3(this XYZ xyz) {
+    public static class RevitExtensions
+    {
+        public static Vector3 ToVector3(this XYZ xyz)
```

Comment:
We can decide if we want to add the scaling into this conversion. This is great if it's a point, but not so great if it's considered as a vector, so you could make it optional like `ToVector3(this XYZ xyz, bool scaleToMeters = false)`. This would then simplify a lot of the code below where you're scaling stuff.

Reply:
that sounds good to me.